.github/workflows/pr_build.yml

```diff
@@ -40,12 +40,31 @@ env:
   RUST_VERSION: stable
 
 jobs:
+  linux-test-rust:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        is_push_event:
+          - ${{ github.event_name == 'push' }}
+      fail-fast: false
+    name: ${{ matrix.os }}/rust
+    runs-on: ${{ matrix.os }}
+    container:
+      image: amd64/rust
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Rust & Java toolchain
```

Comment:
The name still has `Java`

Reply:
Yes, we have one builder that installs both Rust and Java. I did not change that. I can change the name here though.

Reply:
I have now created separate actions that just install Rust.